Readability.js

```diff
@@ -1266,23 +1270,25 @@ Readability.prototype = {
     e = e || this._doc;
     if (!e)
       return;
-    var cur = e.firstChild;
 
-    // Remove any root styles, if we're able.
-    if (typeof e.removeAttribute === 'function' && e.className !== 'readability-styled')
-      e.removeAttribute('style');
-
-    // Go until there are no more child nodes
-    while (cur !== null) {
-      if (cur.nodeType === cur.ELEMENT_NODE) {
-        // Remove style attribute(s) :
-        if (cur.className !== "readability-styled")
-          cur.removeAttribute("style");
+    if (e.nodeType === e.ELEMENT_NODE &&
+        e.namespaceURI === 'http://www.w3.org/1999/xhtml' &&
```

Comment:
It seems to call it on the `Document`, so the first instance may not be an `Element` proper, maybe?

Inline SVG I'm mostly worried about `style`, since most of SVG's attributes can be set via CSS as well.